src/sql/workbench/contrib/editData/browser/gridActions.ts

```diff
@@ -46,6 +47,7 @@ export class GridActionProvider {
 		const actions: IAction[] = [];
 		actions.push(this._instantiationService.createInstance(SaveResultAction, SaveResultAction.SAVECSV_ID, SaveResultAction.SAVECSV_LABEL, SaveFormat.CSV, this._dataService));
 		actions.push(this._instantiationService.createInstance(SaveResultAction, SaveResultAction.SAVEJSON_ID, SaveResultAction.SAVEJSON_LABEL, SaveFormat.JSON, this._dataService));
+		actions.push(this._instantiationService.createInstance(SaveResultAction, SaveResultAction.SAVEMARKDOWN_ID, SaveResultAction.SAVEMARKDOWN_LABEL, SaveFormat.MARKDOWN, this._dataService));
```

Comment:
not sure how other providers (e.g. pgsql, mysql, oracle) handle the unknown format scenario. we should add an attribute `supportedExportFormats` to the connection provider declaration. and use it to decide whether an export action should be added here or not.

Reply:
Sounds like a good thing to file as a separate issue - that kind of work is a bit out of the scope of this PR since it's already an issue with the existing functionality.